charmcraft/helptexts.py

```diff
@@ -82,207 +59,223 @@ def _build_item(title, text, title_space):
     return result
 
 
-def get_full_help(command_groups, global_options):
-    """Produce the text for the default help.
-
-    - command_groups: list of grouped commands, as it's defined in the main
-      module
-
-    - global_options: options defined at charmcraft level (not in the commands),
-      with the (options, description) structure
-
-    The help text has the following structure:
+class Helper:
```

Comment:
I would prefer just
```suggestion
class Help:
```

or HelpBuilder, HelpConstructor

Reply:
HelpBuilder it is